src/views/studio/StudioLayout.tsx

```diff
@@ -50,6 +71,31 @@ const StudioLayout = () => {
   const memberSet = accountSet && !!activeMemberId && hasMembership
   const channelSet = memberSet && !!activeChannelId && hasMembership
 
+  // display snackbar when video upload is complete
+  useEffect(() => {
+    if (videoAssets.length) {
+      videoAssets.forEach((video) => {
+        const videoObject = videoAssetsRef.current.find(
+          (videoRef) => videoRef.uploadStatus !== 'completed' && videoRef.contentId === video.contentId
+        )
+        if (videoObject && video.uploadStatus === 'completed') {
+          const file = assetsFiles.find((asset) => asset.contentId === video.contentId)
+          displaySnackbar({
+            customId: video.contentId,
+            title: 'Video ready to be reviewed',
+            description: (file?.blob as File).name,
+            iconType: 'success',
+            timeout: UPLOADED_SNACKBAR_TIMEOUT,
+            actionText: 'See on Joystream',
+            actionIcon: <SvgGlyphExternal />,
+            onActionClick: () => window.open(`/video/${video.parentObject.id}`, '_blank'),
+          })
+        }
+      })
+      videoAssetsRef.current = videoAssets
+    }
+  }, [assetsFiles, displaySnackbar, navigate, videoAssets])
+
```

Comment:
Not 100%, but I think we could just move this snackbar to uploadManager? It will be cleaner keep all this logic together